### ISSUE TITLE
[Automatic Configuration] First package loading does not suggest to install miniconda anymore

### ISSUE THREAD

**Issue by astamm** [rstudio/reticulate#1488]
In the [Automatic Configuration](https://rstudio.github.io/reticulate/articles/python_dependencies.html#automatic-configuration) section of the *Managing an R Package’s Python Dependencies* vignette, it says that:

> Unless the user has explicitly instructed reticulate to use an existing Python environment, reticulate will prompt the user to download and install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) (if necessary).

which was the behaviour I indeed used to see if no miniconda install was detected on my machine.

However now, the following behaviour occur.

If I execute the following code:
```{r}
# install.packages("remotes")
# remotes::install_github("lmjl-alea/rgudhi")
library(rgudhi)
n <- 10
X <- seq_circle(n)
ac <- AlphaComplex$new(points = X)
```

then it triggers:
```
Would you like to create a default python environment for the reticulate package? (Yes/no/cancel)
```
while previously it used to propose to trigger `reticulate::install_miniconda()`.

If I do
```{r}
reticulate::py_config()
```

it triggers the same question as well, suggesting it did not detect (or at least set up) an existing Python environment either.

I am running :
- on Mac OS (2021) Ventura 13.6 with Apple M1 Pro chip.
- R version: 4.3.1
- RStudio version: 2023.09.0 Build 463

Thanks in advance for your help.

**Comment by mkoohafkan** [comment]
I also ran into this issue, and clicking "yes" in the popup did not solve it as it would try (and fail) to create a virtualenv instead. I solved it by manually creating a conda environment named "r-reticulate". It might be better to just throw an error telling the user to create a default environment named "r-reticulate" using their preferred environment manager.

**Comment by Ray569743422** [comment]
Have you tried no ？

**Comment by t-kalinowski** [comment]
Beginning with reticulate version 1.31, reticulate no longer prompts users to install miniconda. Instead, if user has expressed no preference for a Python installation, reticulate will fall back to prompting users to create a default venv named "r-reticulate".
 
See here for a description of how reticulate selects which python installation to bind to, and how you can configure or customize the behavior: https://rstudio.github.io/reticulate/articles/versions.html

**Comment by t-kalinowski** [comment]
That section is only applicable if `Config/reticulate/autoconfigure:` is in a loaded packages DESCRIPTION file.

Are you loading a package with an autoconfigure field in DESCRIPTION?

**Comment by astamm** [comment]
Yes, I probably should have mentioned that in the original message associated with the issue.
